src/eth/primitives/rs.rs

```diff
@@ -0,0 +1,36 @@
+use ethereum_types::U256;
+use sqlx::database::HasValueRef;
+use sqlx::error::BoxDynError;
+
+use crate::gen_newtype_from;
+
+// Type representing `r` and `s` variables
+// from the ECDSA signature
+pub struct Rs(U256);
```

Comment:
Consider if naming it can be more descriptive `EcdsaRs`.

Reply:
makes sense